.github/workflows/validate.yml

```diff
@@ -396,7 +337,11 @@ jobs:
     if: github.ref == 'refs/heads/master'
 
     # IMPORTANT! Any job added to the workflow should be added here too
-    needs: [validate, validate-old-ghcs, build-alpine, dogfooding]
+    # (This doesn't appear to be true, or we'd have merges with failed
+    # bootstrap tests. The pre-merge CI check is sufficient, but we do
+    # need validate and build-alpine to save the corresponding artifacts.
```

Comment:
This implies that dog fooding is so keep not necessary? I'd strike it. 

Remember: we'd want to lean on in the side of uploading the binaries more often than less: this gives the ability to try did that failed locally.

Reply:
I think I want comments from others first.

Reply:
Also, again, CI would need to have passed before the PR gets merged and we reach that point (the head release only happens on `master`, not the PR branch), so I think dogfooding will have passed regardless.

Reply:
In any case, `dogfooding` was doing nothing there, so I've removed it.

Reply:
Thanks!